### ISSUE TITLE
Request to Clean Up Tooltips

### ISSUE THREAD

**Issue by Gogo1951** [papa-smurf/Gargul#40]
![image](https://user-images.githubusercontent.com/55365231/164091184-4f20df49-70cf-48ec-a411-a025e8fd6532.png)

Ok, I see the tooltips now. (=

They should get the same "branding" treatment that the raid messages do.

{rt3} Gargul : Soft Reserved by {Name}

or

{rt3} Gargul : Soft Reserved by {Name}, {Name}, and {Name}

or

{rt3} Gargul : No Reserves

Make the colors white, class colors for names are fine but the pink/non-white text kinda messy. Tool-tip text should not be split into 3 lines.

Also good to include some helper text, "{rt4} Gargul : Alt + Shift + Left Click to Open Loot Window" something like that.

Good to show the tool-tip only on gear that is still tradable.

(Admittedly my tool-tips are cluttered, I need to turn off the non-TSM Sell Price shit.)

**Comment by papa-smurf** [comment]
Some good ideas, but this part is not up to me and seems to be one of your add-ons throwing some juicy stuff into the mix: `Tool-tip text should not be split into 3 lines.`.

**Comment by papa-smurf** [comment]
(unless you can see something wrong with this line: `Tooltip:AddLine(string.format("\n|cFFEFB8CD%s|r", "Reserved by"));`)

**Comment by Gogo1951** [comment]
Something about `GameTooltipTextLeft` vs `Tooltip:AddLine` and adding a block for your add-on in the tool-tip. Been a while since I played with this. I do remember it took a bit of trial and error to get this to work right in Questie and GogoLoot when we built in the creator names in tooltips.

![image](https://user-images.githubusercontent.com/55365231/164912134-095f587f-8749-4273-a2c2-27d5ed2ff48b.png)
